dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/LogStrategyTest.kt

```diff
@@ -53,8 +56,8 @@ import org.mockito.junit.jupiter.MockitoSettings
 @MockitoSettings()
 internal abstract class LogStrategyTest {
```

Comment:
I'm thinking maybe we could create an abstract "PersistingStrategy" class as a generic version of LogStrategy. That way the LogStrategy itself would only be passing the LogSerializer, and testing that class would mean that you don't care how to serialize (and deserialize for the test) logs, and just care that data is written and read exactly as is. SRP :)